x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas.test.ts

```diff
@@ -4,12 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { createRulesSchema, updateRulesSchema, findRulesSchema, queryRulesSchema } from './schemas';
+import {
```

Comment:
I think after this PR we should do a fast follow up PR where we get together and break apart some of these larger files into more folder structures now that they have grown so much over a zoom chat.

For example, this one would be broken into:

```ts
routes/schema/create_rules_schema.test.ts
routes/schema/update_rules_schema.test.ts
...
```

And the schema might still live in just:

```ts
routes/schema/schemas.ts
```

but at least putting them all into something like that would be more helpful now that we have grown the amount of features so much on the backend. 

For alerts folder and routes folder (again after this PR IMHO) ... just a pure movement/cleanup PR ...

we would do something like:

```ts
routes/index
routes/rules
routes/signals
```

Reply:
That sounds good to me 👍